docs/checkers/param-duplication.md

```diff
@@ -0,0 +1,13 @@
+## param-duplication
```

Comment:
I think adding this header is redundant as filename identifies checker/rule name uniquely.
And this file alone is not a complete reference, only partial documentation.